src/main/java/ro/hiringsystem/model/enums/InterviewerType.java

```diff
@@ -0,0 +1,20 @@
+package ro.hiringsystem.model.enums;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+import java.util.Optional;
+
+@Getter
+@AllArgsConstructor
+public enum InterviewerType {
```

Comment:
Enums already provide a name() method so it would be redundant to actually store a parameter with the same value but lowercase. The parameter should be either removed or replaced by something else like a description.

Reply:
I removed the name field from UserType enum (as I thought a description of the user roles would be redundant there as well), but I added a description field to the InterviewerType enum (for defining the roles of the two categories of interviewers, in the assessment process). Let me know if you agree with this approach.

Reply:
Yes, sounds good to me.